File: app/src/main/java/com/snaptiongame/app/data/models/Tag.java
package com.snaptiongame.app.data.models;

import com.google.gson.annotations.SerializedName;

/**
 * @author Tyler Wong
 */

public class Tag {

    @SerializedName(ID)
    public int id;
    @SerializedName(NAME)
    public String name;

    public static final String ID = "id";
    public static final String NAME = "name";
}

File: app/src/main/java/com/snaptiongame/app/data/models/Tag.kt
package com.snaptiongame.app.data.models

import com.google.gson.annotations.SerializedName

/**
 * @author Tyler Wong
 */

data class Tag (@SerializedName(ID) var id: Int = 0,
                @SerializedName(NAME) var name: String? = null) {
    companion object {
        const val ID = "id"
        const val NAME = "name"
    }
}
